src/components/aeIdentity/aeIdentity.scss

```diff
@@ -0,0 +1,139 @@
+@import "./../variables";
+
+.ae-identity {
+  top:auto;
+  bottom:10px;
+  box-sizing:border-box;
+  zoom: 1.1;
+  padding:15px 21px;
+  color:white;
+  z-index:5;
+  margin:0 auto;
+  text-align: left;
+  width: 335px;
+  box-shadow: 0 4px 8px 0 rgba(60, 60, 60, 0.1);
+  border-radius: 12px;
+  background-image: linear-gradient(122deg, #6f53a1, $exciting);
+
+  &.active{
+    background-image: linear-gradient(119deg, #f22f70, #42255c);
+  }
+
+  &.collapsed{
+  }
+
+  .footer {
+    margin-top: 18px;
+
+    .buttons {
+      position: relative;
+      width: 100%;
+      display: flex;
+      flex-wrap: wrap;
+      align-items: right;
+      justify-content: flex-end;
+      margin-top: 12px;
+    }
+
+    .ae-button {
+      margin-left: 5px;
+    }
+  }
+
+
+  .avatar {
+    width: 23px;
+    height: 23px;
+
+    &.ae-identity-avatar {
+      margin-bottom: 0px;
+    }
+  }
+
+  .flex-row{
+    position:relative;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .chunked-address {
+    display: flex;
+    flex-wrap: wrap;
+    margin-top: 10px;
+    margin-left: -10px;
+
+    .chunk {
```

Comment:
it's hard to understand the context of the `.chunk` on a smaller screen
please don't indent fore than two times. better write 
```
  .chunked-address .chunk {
...
```